src/components/group-feed-type-picker.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+export default class GroupFeedTypePicker extends React.Component {
+
+  handleChange = (property) => (event) => {
+    this.props.updateGroupPrivacySettings({ [property]: event.target.value })
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="form-group">
+          <div className="left-block">
+            <label>Who can view posts:</label>
```

Comment:
There's no noun "post" (at least with the meaning "piece of text") in English. FriendFeed used the term "entry" for this, which we are using as well. (Not in the code, but I'm gradually fixing this in the code as well, see the commit message in 210e561.) So this one should read `Who can read group entries:`.